doc/ocpp_201_README.md

```diff
@@ -0,0 +1,466 @@
+| `set_charging_profiles_callback`                        | Indicates when a charging profile is received and accepted. |
+| `stop_transaction_callback`                             | Used to stop a transaction. Called when the idTagInfo.status of a StartTransaction.conf is not Accepted, when a RemoteStopTransaction.req is received, or when an UnlockConnector.req is received. |
+| `time_sync_callback`                                    | Called on boot notification if the TimeSource ControllerComponent contains Heartbeat. |
+| `transaction_event_callback`                            | Called when a transaction_event was sent to the CSMS. |
+| `transaction_event_response_callback`                   | Called when a transaction_event_response was received from the CSMS. |
+| `unlock_connector_callback`                             | Used by libocpp to force unlock a connector |
+| `update_firmware_request_callback`                      | Initiates a firmware update request. Triggers a security event notification if the certificate is Invalid or Revoked. |
+| `validate_network_profile_callback`                     | Validates the submitted Network Profile. Is Rejected if:<br>• No callback registered to validate network profile<br>• CSMS attempted to set a network profile with a lower securityProfile<br>• CSMS attempted to set a network profile that could not be validated<br>• Network profile could not be written to the device model storage<br> |
+| `variable_changed_callback`                             | Called when a variable has been changed by the CSMS |
+
+### Event Handlers to Call
+Event handlers to call are found in the [ChargePoint](https://github.com/EVerest/libocpp/blob/2124d055f9cc8f253d3a2441ec2d70686174d0ed/include/ocpp/v201/charge_point.hpp#L116).
+
+### Initialize the Database
+
+Use provided [SQLite database](/doc/database_migrations.md) or implement your own storage drive.
+
+### OCPP 2.0.1 Use Cases
+
+| Clause | Use Case Name                     |
+| :----- | :-------------------------------- |
+| A      | Security                          |
+| B      | Provisioning                      |
+| C      | Authorization                     |
+| D      | LocalAuthorizationList Management |
+| E      | Transactions                      |
+| F      | RemoteControl                     |
+| G      | Availability                      |
+| H      | Reservation                       |
+| I      | TariffAndCost                     |
+| J      | MeterValues                       |
+| K      | [SmartCharging](#smart-charging)  |
+| L      | FirmwareManagement                |
+| M      | ISO 15118 CertificateManagement   |
+| N      | Diagnostics                       |
+| O      | DisplayMessage                    |
+| P      | DataTransfer                      |
```

Comment:
```suggestion
| Functional Block | Subject.                                    |
| :--------------- | :------------------------------------------ |
| A                | Security                                    |
| B                | Provisioning                                |
| C                | Authorization                               |
| D                | Local Authorization List Management         |
| E                | Transactions                                |
| F                | Remote Control                              |
| G                | Availability                                |
| H                | Reservation                                 |
| I                | Tariff and Cost                             |
| J                | Meter Values                                |
| K                | [Smart Charging](#smart-charging-use-cases) |
| L                | Firmware Management                         |
| M                | ISO 15118 Certificate Management            |
| N                | Diagnostics                                 |
| O                | Display Message                             |
| P                | Data Transfer                               |
```

Reply:
I'm not really sure this table is necessary anymore. It feels like a _non sequitur_ to me.